test/cluster_SUITE.erl

```diff
@@ -1083,15 +1163,20 @@ start_n_nodes(NamePrefix, Count) ->
 -if(?OTP_RELEASE >= 25).
 start_erlang_node(Name) ->
     Name1 = list_to_atom(Name),
-    {ok, _, Node} = peer:start(#{name => Name1,
+    {ok, Peer, Node} = peer:start(#{name => Name1,
                                  wait_boot => infinity}),
-    Node.
+    {Node, Peer}.
+stop_erlang_node(_Node, Peer) ->
+    ok = peer:stop(Peer).
 -else.
 start_erlang_node(Name) ->
     Name1 = list_to_atom(Name),
     Options = [{monitor_master, true}],
     {ok, Node} = ct_slave:start(Name1, Options),
     Node.
```

Comment:
This copy of `start_erlang_node/1` for Erlang<25 does not return the same thing as the copy for Erlang>=25.

It could return `{Node, Node}` where the second argument is an opaque value required by the stop function.